source/hwPortUtils.py

```diff
@@ -93,6 +93,16 @@ class dummy(ctypes.Structure):
 SetupDiGetDeviceRegistryProperty.argtypes = (HDEVINFO, PSP_DEVINFO_DATA, DWORD, PDWORD, ctypes.c_void_p, DWORD, PDWORD)
 SetupDiGetDeviceRegistryProperty.restype = BOOL
 
+SetupDiEnumDeviceInfo = ctypes.windll.setupapi.SetupDiEnumDeviceInfo
+SetupDiEnumDeviceInfo.argtypes = (HDEVINFO, DWORD, PSP_DEVINFO_DATA)
+SetupDiEnumDeviceInfo.restype = BOOL
+
+CM_Get_Device_ID = ctypes.windll.cfgmgr32.CM_Get_Device_IDW

```

Comment:
@Jcsteh, it seems either you added this in your initial work, or it has been removed from core and I accidentally added it again. Anyway, it is not used within hwPortUtils. I just want to make sure what you intended to do with this function.